core/producers/trigAprProducers.fcl

```diff
@@ -53,7 +53,10 @@ TrigAprProducers : {
   aprLowPStopTargTriggerInfoMerger: {
     module_type: "MergeTriggerInfo"
   }
-  aprLowPStopTargMultiTrkTriggerInfoMerger: {
+  aprHighPStopTargMultiTrkTriggerInfoMerger: {
+    module_type: "MergeTriggerInfo"
+  }
+  aprLowPStopTargMultiHelixTriggerInfoMerger: {
```

Comment:
To match your later usage, the low momentum trigger info merger name should probably be `aprLowPMultiHelixTriggerInfoMerger`.